static/styles/zulip.css

```diff
@@ -1388,10 +1388,127 @@ blockquote p {
     font-weight: bold;
 }
 
+.alert-box {
```

Comment:
Yeah, I would err on the side of fixing everything here, since whatever preceded you in the "alerts" arena probably wasn't super well thought out, and it would be good to have you make everything consistent.